performance_benchmarks/generate_figures.py

```diff
@@ -57,6 +57,7 @@ def extract_data(req_ps, req_served, benchmarks):
 }
```

Comment:
In this dict `CherryPy` is actually `CherryPy+\nCheroot`.

Reply:
I removed the CherryPy benchmark